concourse/scripts/sleep_for_deploy_env.sh

```diff
@@ -2,14 +2,15 @@
 
 set -eu
 
+splay=$((60*60))
 # The following sleep monstrosity deterministically sleeps for a
-# period of time between 0-20mins in order to prevent all our
+# period of time between 0-${splay} seconds in order to prevent all our
```

Comment:
Is that seconds afterall?

Reply:
It is now, because it's set to 60*60 = 3600